drkafka/config/doctorkafka.properties

```diff
@@ -42,54 +61,125 @@ doctorkafka.brokerstats.consumer.ssl.truststore.location=truststore_path
+## ...
 
-# [required] doctorkafka web port
-doctorkafka.web.port=8080
+## Interval timer
+###
+doctorkafka.monitors.interval_timer.class=com.pinterest.doctorkafka.modules.monitor.IntervalTimer
+doctorkafka.monitors.interval_timer.config.interval.seconds=5
 
-# [optional] disable doctorkafka service restart
-doctorkafka.restart.disabled=false
+## Maintenance mode checker
+doctorkafka.monitors.maintenance_mode.class=com.pinterest.doctorkafka.modules.monitor.cluster.MaintenanceChecker
 
-# [required] doctorkafka service restart interval
-doctorkafka.restart.interval.seconds=86400
+## Dead broker monitor
+doctorkafka.monitors.dead_broker_monitor.class=com.pinterest.doctorkafka.modules.monitor.cluster.kafka.DeadBrokerMonitor
+doctorkafka.monitors.dead_broker_monitor.config.no_stats.seconds=600
 
-# [optional] ostrich port #
-doctorkafka.ostrich.port=2052
+## No stats broker monitor
+doctorkafka.monitors.no_stats_broker_monitor.class=com.pinterest.doctorkafka.modules.monitor.cluster.kafka.NoStatsBrokerMonitor
 
-# [optional] tsd host and port.
-doctorkafka.tsd.hostport=localhost:18621
+## URP monitor
+doctorkafka.monitors.urp_monitor.class=com.pinterest.doctorkafka.modules.monitor.cluster.kafka.URPMonitor
 
-# [required] email addresses for sending general notification on cluster under-replication etc.
-doctorkafka.emails.notification=email_address_1,email_address_2
+# default plugins if no plugins are specified for a cluster. Plugins will be evaluated in the same order as listed
 
-# [required] email addresses for sending alerts to
-doctorkafka.emails.alert=email_address_3,email_address_4
 
-# [optional] brokerstats.version
-doctorkafka.brokerstats.version=0.2.4.8
+# operator configs
+## BrokerReplacer config
+### [required]
+doctorkafka.operators.broker_replacer.class=com.pinterest.doctorkafka.modules.operator.cluster.kafka.BrokerReplacer
+### [optional]
+doctorkafka.operators.broker_replacer.config.replacement.cooldown.seconds=43200
 
+## NoStatsBrokersOperator config
+### [required]
+doctorkafka.operators.no_stats_brokers_operator.class=com.pinterest.doctorkafka.modules.operator.cluster.kafka.NoStatsBrokersOperator
+
+## URPReassignor config
+### [required]
+doctorkafka.operators.urp_reassignor.class=com.pinterest.doctorkafka.modules.operator.cluster.kafka.URPReassignor
+# please override per cluster based on fleet capacity
+doctorkafka.operators.urp_reassignor.config.network.bandwidth.max.mb=200
+### [optional]
+doctorkafka.operators.urp_reassignor.config.rack.awareness.enabled=false
+doctorkafka.operators.urp_reassignor.config.prolong.urp.alert.seconds=7200
+
+
+# action module configs
+#
+# actions will listen to subscribed events and act whenever it receives an event
+#
+# actions are in dry run mode by default
+# disable dry run mode by setting dryrun=false explicitly
+
+## report_operation action
+### [required]
+doctorkafka.actions.report_operation.class=com.pinterest.doctorkafka.modules.action.KafkaReportOperation
```

Comment:
is `com.pinterest.doctorkafka.modules.action.KafkaReportOperation` the default value that the users can put?  shall we clarify that it is the default value that users can use?